internal/cmd/options.go

```diff
@@ -10,6 +10,9 @@ type Options struct {
 	Env          Env
 	Stderr       io.Writer
 	MutateConfig func(*config.Config)
+	// TODO: Move these to a command-specific struct
+	Tags    []string
+	Against string
```

Comment:
This is gross, right now we expect the syntax of the `Verify` and `Push` command to be the same. Will clean this up in another pull request.